pre-workshop/pre-workshop-ja.md

```diff
@@ -8,182 +8,178 @@
 
+本研究の参加者としての権利や待遇について質問がある場合、または本研究について意見を提供したい場合は、カリフォルニア大学デービス校のInstitutional Review Board（IRB）（電話：(916) 703-9151、IRBAdmin@ucdmc.ucdavis.edu、または2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817）までご連絡ください。
 -----------
 
-### Do you consent to taking this survey? *
-
-   - yes
-   - no
-
-### Are you 18 years of age or above? *
-
-   - yes
-   - no
-
-### What is the identifier for your workshop? It is in the "YYYY-MM-DD-sitename" format and is used in the URL for the workshop website or the Etherpad you used.  Please copy and paste the workshop identifier into the text box below.  *
-
-(short text answer)
-
-### Which of the following workshops are you attending? 
-
-- Data Carpentry Ecology 
-- Data Carpentry Economics 
-- Data Carpentry Genomics 
-- Data Carpentry Geospatial 
-- Data Carpentry Social Sciences 
-- Library Carpentry 
-- Software Carpentry 
-- I don’t know 
-
-### Please enter a unique identifier as follows: Number of siblings (as numeric) + First two letters of the city you were born in (lowercase) + First three letters of your current street (lowercase).  
-
-(short text answer)
-
-### Please indicate your relevant fields or disciplines.  
-
-- Agricultural or Environmental Sciences 
-- Biomedical or Health Sciences 
-- Chemistry 
-- Civil, Mechanical, Chemical, or Nuclear Engineering 
-- Computer Science or Electrical Engineering 
-- Earth Sciences 
-- Economics or Business 
-- Education 
-- Genetics, Genomics, or Bioinformatics 
-- High Performance Computing 
-- Humanities 
-- Library and Information Science 
-- Life Sciences 
-- Mathematics or Statistics 
-- Medicine 
-- Organismal Biology (Ecology, Botany, Zoology, Microbiology, etc.) 
-- Physical Sciences 
-- Planetary Sciences (Geology, Climatology, Oceanography, etc.) 
-- Psychology or Neuroscience 
-- Social Sciences 
-- Space Sciences 
-
-### What is your current occupation/career stage?  
-
-- Administrative Staff 
-- Analyst 
-- Commercial Software Developer 
-- Faculty 
-- Government Employee 
-- Graduate Student 
-- Industry Employee 
-- Librarian or Archivist 
-- Management 
-- Postdoctoral Researcher 
-- Research Staff (including Researcher Programmer) 
-- Retired or Not Employed 
-- Support Staff (including Technical Support) 
-- Undergraduate Student 
-
-### What operating system is on the computer you are using at the workshop? 
-
-- macOS 
+### このアンケートを受けることに同意しますか？ *
+
+   - はい
+   - いいえ
+
+### 18歳以上の方ですか？ *
+
+   - はい
+   - いいえ
+
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename 形式で、ワークショップのウェブサイトまたは使用した Etherpad の URL に使用されています。以下のテキストボックスにワークショップ ID をコピーして貼り付けてください。
+(短文による回答)
+
+### どのワークショップに参加しますか？ 
+
+- データカーペントリー エコロジー 
+- データカーペントリー 経済学 
+- データカーペントリー ジェノミクス 
+- データカーペントリー 地理学 
+- データカーペントリー 社会科学 
+- ライブラリカーペントリー 
+- ソフトウェアカーペントリー 
+- 分からない
+
+### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (小文字)
```

Comment:
```suggestion
### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (ローマ字小文字)。
```

Update for consistency with #4. @naganeayu suggested to specify latin alphabet inputs to ensure alphanumeric IDs without CJK characters (e.g., Kanji, Kana).